### PR TITLE
Only show standards applicable to land values entered

### PR DESCRIPTION
**Pull request by paulsimonandrews**
https://eaflood.atlassian.net/browse/SF-250

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfi-frontend",
-  "version": "1.17.1",
+  "version": "1.17.2",
```
Comment (st3v3nhunt) [inline package.json]: A bit more than a bug fix I reckon!

File: app/views/land-values.njk [hunk 2]
```diff
@@ -28,21 +27,33 @@
...
-    <h1 class="govuk-heading-l">{{title}}</h1>
 
-    <form method="POST" autocomplete="off" novalidate >
-      {% for standard in components.inputs %}
-        {{ govukInput(standard) }}
-      {% endfor %}
 
-      {{ govukDetails(components.details) }}
+    <form method="POST" autocomplete="off" novalidate >
+      {% if errorText %}
```
Comment (st3v3nhunt) [inline app/views/land-values.njk]: This `if` block could be simplified to
`<div class="govuk-form-group {{ "govuk-form-group--error" if errorText }}">`

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
Looks good, I like the reduction in duplication for the standards.

File: app/views/land-values.njk (hunk 2 above)
Comment (paulsimonandrews) [reply app/views/land-values.njk]: Good spot. Fixed.

File: package.json (hunk 1 above)
Comment (paulsimonandrews) [reply package.json]: Indeed. The intention was for it to be smaller! Updated to `1.18.0`

**Review by st3v3nhunt — APPROVED** [review]
LGTM 👍

**Pull request merged**